.github/workflows/run-e2e.yml

```diff
@@ -39,7 +39,6 @@ jobs:
         run: cd src/e2e && npx playwright install --with-deps
       - name: Run your tests
         run: 'cd src/e2e && npm run e2e:run -- --world-parameters "{\"urls\": { \"zac\": \"${{ secrets.ZAC_URL }}\", \"openForms\": \"${{ secrets.OPEN_FORMS_URL }}\" }, \"headless\": true, \"users\": { \"Bob\": { \"username\": \"${{ secrets.ZAC_E2E_TEST_USER_1_USERNAME }}\", \"password\": \"${{ secrets.ZAC_E2E_TEST_USER_1_PASSWORD }}\"}, \"Oscar\": { \"username\": \"${{ secrets.ZAC_E2E_TEST_USER_2_USERNAME }}\", \"password\": \"${{ secrets.ZAC_E2E_TEST_USER_2_PASSWORD }}\"}}}"'
-        continue-on-error: true
```

Comment:
The fix is to remove this line? Not sure what it does but seems to me we do want to continue running the remaining tests if there is an error?

Reply:
This one does not change the state of the build to error though. So the result sent is success :(